src/Guest.java

```diff
@@ -1,6 +1,19 @@
 public class Guest extends Person {
 
-    public Guest(String name) {
-        super(name);
+    public Guest(String name, double temperature, double bloodAlcohol) {
+        super(name, temperature, bloodAlcohol);
+        boolean registrationStatus;
```

Comment:
Idem

Reply:
Had ik erbij verzonnen. Wanneer een gast een zwembad in wil gaan moet hij gecontroleerd worden. Het leek mij handig om bij te houden wanneer de gast zich geregistreerd had.